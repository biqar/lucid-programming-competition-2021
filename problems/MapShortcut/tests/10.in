100
Right 107 78
Right 114 36
Left 13 50
Right 123 28
Left 80 64
Left 61 27
Left 137 12
Left 88 30
Left 171 63
Right 128 36
Right 163 23
Left 150 68
Right 17 12
Left 70 74
Right 120 85
Right 159 25
Right 31 14
Left 92 81
Left 154 63
Left 77 82
Right 46 85
Right 57 6
Left 130 14
Right 145 96
Left 46 15
Right 75 65
Right 51 88
Left 97 79
Left 85 4
Right 135 100
Left 61 77
Left 20 13
Left 127 95
Right 136 71
Left 119 68
Right 78 3
Right 73 53
Left 62 81
Left 2 66
Right 65 20
Left 70 32
Right 178 72
Right 156 10
Right 148 57
Right 94 87
Right 167 84
Right 165 29
Right 93 30
Right 120 71
Left 89 16
Left 130 97
Right 159 46
Left 52 22
Right 120 89
Left 69 42
Left 74 1
Left 5 25
Left 168 57
Right 172 28
Right 100 37
Left 12 38
Left 96 8
Left 62 59
Right 90 38
Right 134 19
Left 4 12
Left 10 77
Left 44 64
Right 39 7
Left 125 19
Left 109 70
Right 138 97
Left 4 71
Right 171 100
Left 66 45
Left 66 40
Left 107 70
Left 43 65
Right 148 56
Left 129 12
Left 49 100
Right 167 69
Left 23 12
Left 106 2
Right 151 79
Right 121 37
Left 67 23
Right 69 17
Left 39 25
Right 43 25
Left 157 53
Right 160 51
Left 72 74
Right 82 31
Right 75 61
Right 80 82
Right 77 60
Right 74 69
Left 116 27
Left 165 41
