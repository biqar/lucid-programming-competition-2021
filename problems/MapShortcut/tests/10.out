Left 66 905
